snap7/common.py

```diff
@@ -29,9 +29,11 @@ class ADict(dict):
 
 
 class Snap7Library:
-    """
-    Snap7 loader and encapsulator. We make this a singleton to make
-    sure the library is loaded only once.
+    """Snap7 loader and encapsulator. We make this a singleton to make
+        sure the library is loaded only once.
+
+    Attributes:
+        lib_location (str, Optional): full path to the `snap7.dll` file. Defaults to None
```

Comment:
If I'm not mistaken, you don't need to include the types in the docstrings since they are taken from type hinting already.

Reply:
You are right.
I was following [this](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) example.
But according to [sphinx documentation](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#type-annotations) is not necessary. I will remove them.